Kernel/Bus/USB/USBManagement.cpp

```diff
@@ -64,6 +65,8 @@ bool USBManagement::initialized()
 
 UNMAP_AFTER_INIT void USBManagement::initialize()
 {
+    // FIXME: This probably shouldn't be done here.
```

Comment:
Maybe we should put a "guard" here to ensure USBManagement was not initialized before so we don't call `USB::SysFSUSBBusDirectory::initialize();` more than once. Anyway, I don't think there's a better place to call to this initialize of SysFS directory if we take proper measures to protect from initializing things multiple times :)

Reply:
Done :+1: